tests/scenarios/siocli_prompt/run.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Copyright (C) 2019-2020 Siodb GmbH. All rights reserved.
+# Use of this source code is governed by a license that can be found
+# in the LICENSE file.
+
+## Global
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TEST_NAME=$(basename "${SCRIPT_DIR}")
+source "${SCRIPT_DIR}/../../share/CommonFunctions.sh"
+
+## Specific test functions
+
+## Specific test parameters
+
+## =============================================
+## TEST HEADER
+## =============================================
+_TestBegin
+_Prepare
+_StartSiodb
+
+## =============================================
+## TEST
+## =============================================
+
+_RunSql "       -- QUERY 1
+   -- just a comment --
+        select
+         -- just a comment
+     name  -- just a comment --
+                           --
+                  from /* just a comment line 1
+             just a comment line 2
+             just a comment line 3 */
+                     sys_databases
+           where --
+         trid = 1
+         --
+         /*
+             just a comment line 1
+             */   ;
+   -- just a comment --
+        select
+         -- just a comment
+     name  -- just a comment --
+                           --
+                  from /* just a comment line 1
```

Comment:
Please add something like this somewhere here and make sure it works:

Case 1:
```sql
/*
    this is a multiline comment
  -- this is attempt to use comment in comment, but this is in fact part of previously started multiline comment
*/ 
```

Case 2:
```sql
/*
    this is a multiline comment
  -- this is attempt to use comment in comment, but this is in fact part of previously started
-- multiline comment which ends right here*/ 
```

Case 3:
```sql
-- this is single line comments  /* but NOT a multiline comment despite /*
```
Case 4:
```
/* I start multline comment /* multiple /* times
but that /* should /* not /* matter  */
```

Reply:
## Case 1: OK

```sql
siocli>     select *
      > /*
      >     this is a multiline comment
      >   -- this is attempt to use comment in comment, but this is in fact part of previously started multiline comment
      > */
      >    from
      >           /*
      >               this is a multiline comment
      >             -- this is attempt to use comment in comment, but this is in fact part of previously started multiline comment
      >           */
      >           sys_dummy
      >           ;

TRID                DUMMY
------------------- -----------

Row(s): 0

Elapsed time: 25 ms.

```

## case 2: OK

```sql
siocli> /*
      >     this is a multiline comment
      >   -- this is attempt to use comment in comment, but this is in fact part of previously started
      > -- multiline comment which ends right here*/
      > select * from sys_dummy ;

TRID                DUMMY
------------------- -----------

Row(s): 0

Elapsed time: 10 ms.
```

## Case 3 & 4:  OK

```sql
select * 
              /* I start multline comment /* multiple /* times
                 but that /* should /* not /* 
                 matter  
              */
from
-- this is single line comments  /* but NOT a multiline comment despite /*
sys_dummy 
-- this is single line comments  /* but NOT a multiline comment despite /*
;
```

This is because most of the comments are parsed by the parser. But thanks to your feedback I discovered that a multiline comment out of a query does not work properly and that we must cover this case. I'll work on it.

Nonetheless, I'll add those tests to testing scripts.

Reply:
From the above it is a bit not clear to me whether you are planning or not to solve this in this PR or that is already solved. If something doesn't work and you don't plan to fix it here, please create new issue, this will not let us completely forget about it. Anyway, meanwhile I'll request couple more changes so you have a chance to include that, if you are not have done yet, otherwise, please create new issue.

Reply:
I meant that I will have to fix it in the PR and it's now fixed since the last PR review you did.